database/migrations/sql/20180604_04_create_organisations.sql

```diff
@@ -75,6 +75,17 @@ CREATE TABLE community_business (
   CONSTRAINT community_business_to_community_business_region_fk FOREIGN KEY (community_business_region_id) REFERENCES community_business_region
 );
 
+CREATE TABLE volunteer_admin_code (
+  volunteer_admin_code_id SERIAL NOT NULL UNIQUE,
+  organisation_id                INT NOT NULL,
```

Comment:
Do we want to support multiple admin codes per organisation? If not this should be `UNIQUE`

Reply:
Previously I thought these were 5 digit numerical values and didn't want to restrict ourselves to 99999. But I was wrong, so this can go back in 😊

Reply:
Oh wait, I though this was on the `code` column. Do we want this to be unique as well?